### PR TITLE
Python3.5 in tox tests and raise vs raise e

### PR DESCRIPTION
**Pull request by w-vi**
Just a small wrap up / cleanup patch

### CODE REVIEW

File: dredd_hooks/dredd.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 #
-#  Copyright (c) 2015 Vilibald Wanča
+#  Copyright (c) 2015, 2016 Vilibald Wanča
```
Comment (kylef) [inline dredd_hooks/dredd.py]: Shouldn't the copyright be Apiary?

Comment (w-vi) [inline dredd_hooks/dredd.py]: I guess it should ...

**Comment by w-vi** [comment]
@kylef check the copyright once more please

**Comment by kylef** [comment]
@w-vi :+1:

**Pull request merged**